src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -351,31 +349,31 @@
         "ENABLE_ACCOUNT": "Activer le compte",
         "DELETE": "Supprimer",
         "CREAT_ROBOT_ACCOUNT": "Création d'un compte robot",
-        "PERMISSIONS_ARTIFACT": "Artéfact",
+        "PERMISSIONS_ARTIFACT": "Artefact",
         "PERMISSIONS_HELMCHART": "Helm Chart (Chart Museum)",
         "PUSH": "Pousser",
         "PULL": "Tirer",
```

Comment:
In french, are those terms really used? Maybe it should not be translated

Reply:
Yeah I don't know, I asked myself the same question but as I couldn't take a decision I let it as it was.
Shoudl we keep "push" and "pull" everywhere ?

Reply:
Pull/Push are also used in french, I think we can keep english here.